### PR TITLE
SAK-30476: jquery and jquery-ui have moved so change citations refere…

### PR DESCRIPTION
**Pull request by nicholaswilson100**

### CODE REVIEW

**Comment by buckett** [comment]
I think you should be pointing at the webjars copy now. https://jira.sakaiproject.org/browse/SAK-30278

**Comment by nicholaswilson100** [comment]
thanks

**Comment by nicholaswilson100** [comment]
I've changed the references to the webjar ones

File: citations/citations-servlet/servlet/src/webapp/vm/servlet.vm [hunk 1]
```diff
@@ -20,7 +20,7 @@
 		<title>$tlang.getString( "window.gscholar" )</title>
 		<link href="/library/skin/tool_base.css" type="text/css" rel="stylesheet" media="all" />
 		<link href="/library/skin/default/tool.css" type="text/css" rel="stylesheet" media="all" />
-		<script type="text/javascript" src="/library/js/jquery/1.7.1/jquery-1.7.1.min.js"></script>
+		<script type="text/javascript" src="webjars/jquery/1.11.3/jquery.min.js"></script>
```
Comment (buckett) [inline citations/citations-servlet/servlet/src/webapp/vm/servlet.vm]: Does this work as aren't the webjars only served up from `/library` ?

Comment (nicholaswilson100) [inline citations/citations-servlet/servlet/src/webapp/vm/servlet.vm]: Yes you're right (and I don't think servlet.vm and reorder.vm are used anymore).

**Pull request merged**